ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/intended/configs/port-channel-short-esi.cfg

```diff
@@ -16,12 +16,28 @@ vrf instance MGMT
 interface Port-Channel10
    description server01_ESI_PortChanne1
    no shutdown
-   switchport
+   no switchport
    evpn ethernet-segment
       identifier 0000:0000:0303:0202:0101
       route-target import 03:03:02:02:01:01
    lacp system-id 0303.0202.0101
 !
+interface Port-Channel10.111
+   vlan id 111
+   encapsulation vlan
+      client dot1q 111 network client
+   evpn ethernet-segment
+      identifier 0000:0000:0303:0202:0111
+      route-target import 03:03:02:02:01:11
+!
+interface Port-Channel10.222
+   vlan id 222
+   encapsulation vlan
+      client dot1q 222 network client
+   evpn ethernet-segment
+      identifier 0000:0000:0303:0202:0222
+      route-target import 03:03:02:02:02:22
+!
```

Comment:
@ClausHolbechArista - I am not 100% sure here - is this output correct from an EOS point of view?